README.md

```diff
@@ -239,16 +240,14 @@ After setting up your site on search.gov you can then add your `site handle` to
 You can add Google Analytics to your site by uncommenting the `google_analytics_ua` line and replacing `UA-????????-??` with your Google analytics UA code.
 
 ```
-# Configuration for Google Analytics, add your UA code here:
```

Comment:
Removing some of the extra commented sections here to just get at what need to be commented out